bitnami/rabbitmq/values.yaml

```diff
@@ -284,7 +284,7 @@ configuration: |-
   ## Username and password
   ##
   default_user = {{ .Values.auth.username }}
-  default_pass = CHANGEME
+  default_pass = {{ template "rabbitmq.secretPasswordName" . }}
```

Comment:
What is this ?
Do we really needs this (since we have the load_definition.json below) ?